contracts/Witch.sol

```diff
@@ -64,17 +66,23 @@ contract Witch is AccessControl() {
     /// @dev Set:
     ///  - the auction duration to calculate liquidation prices
     ///  - the proportion of the collateral that will be sold at auction start
+    ///  - the maximum collateral that can be auctioned at the same time
     ///  - the minimum collateral that must be left when buying, unless buying all
+    ///  - The decimals for maximum and minimum
     ///  - whether we are enabling or disabling the ilk
-    function setIlk(bytes6 ilkId, uint32 duration, uint64 initialOffer, uint128 dust, bool active) external auth {
+    function setIlk(bytes6 ilkId, uint32 duration, uint64 initialOffer, uint96 line, uint24 dust, uint8 dec) external auth {
         require (initialOffer <= 1e18, "Only at or under 100%");
         ilks[ilkId] = Ilk({
-            active: active,
             duration: duration,
-            initialOffer: initialOffer,
-            dust: dust
+            initialOffer: initialOffer
+        });
+        limits[ilkId] = Limits({
+            line: line,
+            dust: dust,
+            dec: dec,
+            sum: limits[ilkId].sum
```

Comment:
This is a function to set the limits for an ilk (line, dust) to new values.

`sum` is not a limit, it's the accumulator of the collateral under auction. It's packed with `line` and `dust` because we access it at the same time (same pattern as in the Cauldron with `debt`).

However, when we update `line` or `dust`, which happens via governance, we don't want to change `sum`.